pkg/util/http/transport.go

```diff
@@ -73,11 +73,8 @@ func minTLSVersionFromConfig(cfg config.Config) uint16 {
 	case "tlsv1.3":
 		min = tls.VersionTLS13
 	default:
-		if cfg.GetBool("force_tls_12") {
```

Comment:
We should have a warning that this setting is deprecated and will be removed if we keep it in config.go